craig-rolleston/week4/wed/movie-site/views/layout.erb

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Movie Site</title>
+  <link rel="stylesheet" href="/css/style.css">
+</head>
+
+
+<header>
+<nav> 
+<h1>FIND YOUR MOVIE</h1>
+<hr>
+</nav>
```

Comment:
You can also try put some `a href` links here so users can navigate back to the home page and execute new searches